test/gtest/vec_add.cpp

```diff
@@ -147,9 +151,9 @@ struct VecAddTest : public ::testing::TestWithParam<VecAddTestCase>
 
         std::string params = options.GenerateFor(miopen::kbp::HIP{});
 
-        uint totalElements  = vec_size;
-        int threadsPerBlock = threads_per_block;
-        int blocksPerGrid   = (totalElements + threadsPerBlock - 1) / threadsPerBlock;
+        size_t totalElements = vec_size;
```

Comment:
Several minor but important issues to address: 
* `vecadd_config` and `vec_size, threads_per_block` are duplicating the same info. Keep `vecadd_config` and get rid of the other two members.